packages/react-components/src/molecules/PropertyTaxRegistration.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import PropTypes from "prop-types";
+// import { Card, CardHeader, CardText, LocationSearch, SubmitBar, LinkButton } from "@egovernments/digit-ui-react-components";
+import { useTranslation } from "react-i18next";
+
+import Card from "../atoms/Card";
+import CardHeader from "../atoms/CardHeader";
+import CardText from "../atoms/CardText";
+import CardLabel from "../atoms/CardLabel";
+import SubmitBar from "../atoms/SubmitBar";
+import LinkButton from "../atoms/LinkButton";
+//import box from "../atoms/box";
+
+const PropertyTaxRegistration = ({ goNext, onSelect}) => {
+  return (
+    <Card>
+      <CardHeader>Property Tax Registration</CardHeader>
+
```

Comment:
@Tulika-eGov document required screen is different right, can you remove it if in case not required?